src/static/bootstrap-switch/dist/css/bootstrap-switch.css

```diff
@@ -0,0 +1,196 @@
+/* ========================================================================
```

Comment:
You didn't bring in bootstrap switch because it was git ignored before. Hence why I removed all of the gitignore lines I thought were unused and why I'm adding in the switch lib now. I'll make the changes to still ignore logs and code cover stuff though.